pretext/constants.py

```diff
@@ -2,17 +2,80 @@
 
 NEW_TEMPLATES = ["book", "article", "demo", "hello", "slideshow"]
 
-BUILD_FORMATS = [
-    "html",
-    "pdf",
-    "latex",
-    "epub",
-    "kindle",  # TODO: mode of epub rather than separate format?
-    "braille",
-    "html-zip",  # TODO: deprecate
-    "webwork-sets",  # TODO: just "webwork"
-    "webwork-sets-zipped",  # TODO: deprecate
-]
+FORMATS = ["html", "pdf", "latex", "epub", "kindle", "braille", "webwork", "custom"]
+
+# Give list of assets that each build format requires.
+ASSETS_BY_FORMAT = {
```

Comment:
@rbeezer, when you get a chance, can you check that these are indeed the assets that someone might need to generate for each build format?

Reply:
Those look right to me.  I assume some things are subsumed into others, such as `pg-macros` being part of `webwork`.  `qrcode` part of `interactive`, etc?

Running `pretext/pretext -h` will give you a fairly concise rundown of what core Python can do.

Reply:
Awesome.  Thanks!